collections/Lat_misc/MS_Lat_misc_e_2.xml

```diff
@@ -37,51 +40,215 @@
                      <idno type="SCN">8847</idno>
                   </altIdentifier>
                </msIdentifier>
+               <head>Ars medicinae and other medical texts in Latin and German; 13th century, first half, with additions</head>
                <msContents>
                   <msItem n="1" xml:id="MS_Lat_misc_e_2-item1">
-                     <title key="work_10281">Ars medicine</title>
+                     <locus>(fols. 2r–8r) </locus>   
+                     <author key="person_19690624">Johannitius</author>
+                     <title key="work_2612">Isagoge ad Artem paruum Galeni</title>
+                     <note>(tr. <persName key="person_305230031" role="trl"> Constantinus Africanus </persName>)</note>
+                     <rubric>Incipit Iohannicius liber Isagogarum</rubric>
+                     <incipit><sic>Mmedicina</sic> dividitur in duas partes in theoricam et practicam </incipit>
+                     <explicit>qualitatis. quantitatis. temporis. et ordinis. atque boni malive discrecio</explicit>
+                     <finalRubric>Liber Iohannicii explicit</finalRubric>
+                     <bibl>ed. G. Maurach, ‘Johannicius: Isagoge ad Techne Galieni’, <title>Sudhoffs Archiv</title>, 62 (1978), 148–74.</bibl>
                      <textLang mainLang="la">Latin</textLang>
                   </msItem>
                   <msItem n="2" xml:id="MS_Lat_misc_e_2-item2">
-                     <title key="work_12426">Isagoge ad Artem paruum Galeni</title>
-                     <note>tr. <persName key="person_305230031" role="trl"> Constantinus Africanus </persName>
-                     </note>
+                     <locus>(fols. 8v–‘11-12’v)</locus>
```

Comment:
Do we need to adjust the foliation?

Reply:
We'd have to refoliate the whole volume, so I don't think that would be practical - the '11-12' business is an established Bodleian convention, although many people do dislike / not understand it